tests/arm/checks/resource/test_CosmosDBLocalAuthDisabled.py

```diff
@@ -0,0 +1,41 @@
+import os
+import unittest
+from checkov.runner_filter import RunnerFilter
+from checkov.arm.runner import Runner
+from checkov.arm.checks.resource.CosmosDBLocalAuthDisabled import check
+
+
+class TestCosmosDBLocalAuthDisabled(unittest.TestCase):
+
+    def test(self):
+        runner = Runner()
+        current_dir = os.path.dirname(os.path.realpath(__file__))
+
+        test_files_dir = os.path.join(current_dir, "example_CosmosDBLocalAuthDisabled")
+        report = runner.run(root_folder=test_files_dir,
+                            runner_filter=RunnerFilter(checks=[check.id]))
+        summary = report.get_summary()
+
+        passing_resources = {
+            'Microsoft.DocumentDB/databaseAccounts.pass'
+        }
+        failing_resources = {
+            'Microsoft.DocumentDB/databaseAccounts.fail',
+            'Microsoft.DocumentDB/databaseAccounts.fail1'
+        }
+        skipped_resources = {}
```

Comment:
You add an unknown file in the tests folder but you are not using it in the UT here.

Reply:
The test go to an unknown file and returns CheckResult.UNKNOWN and did not pass or fail why add it to UT?